fakebook/src/app/service/post.service.ts

```diff
@@ -1,38 +1,51 @@
 import { Injectable } from '@angular/core';
-import { HttpClient } from '@angular/common/http';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
 import { Observable } from 'rxjs';
 import { OktaAuthService, OktaCallbackComponent } from '@okta/okta-angular';
 
 import { Post } from '../model/post';
 import { NewPost } from '../model/newpost';
-import { promise } from 'protractor';
 
 @Injectable({
   providedIn: 'root'
 })
 export class PostService {
   constructor(private http: HttpClient, private oktaAuth: OktaAuthService) { }
-  baseUrl = 'someUrl';
-  url = `${this.baseUrl}/api/Posts`; // update with our base url
+  baseUrl = 'https://fakebook.revaturelabs.com/';
```

Comment:
hard-coding this here is not very maintainable. read the value from `environment.ts`, and provide the fakebook value in `environment.prod.ts`. provide whichever value you find useful in `environment.ts`. there better not be tests firing real http requests to any URLs either way.

Reply:
Went and fixed all the Urls that were pointed out, We had dummy ones for production that somehow stuck